modules/mono/csharp_script.cpp

```diff
@@ -1147,6 +1147,7 @@ void CSharpLanguage::_editor_init_callback() {
 	// Add plugin to EditorNode and enable it
 	EditorNode::add_editor_plugin(godotsharp_editor);
 	ED_SHORTCUT("mono/build_solution", TTR("Build Solution"), KeyModifierMask::ALT | Key::B);
+	ED_SHORTCUT_OVERRIDE("mono/build_solution", "macos", KeyModifierMask::META | KeyModifierMask::CTRL | KeyModifierMask::ALT | Key::B);
```

Comment:
You didn't mention this shortcut in the previous table, is it fine to keep it as is or should it be simplified to META+CTRL+B?

Reply:
Indeed, I have missed it. Updated to `META+CTRL+B`.